download_xena_data.py

```diff
@@ -0,0 +1,44 @@
+# This script will download publicly available pancancer data that
+# does not have restricted use
+
+import os
+from urllib import request
+from subprocess import call
+
+if not os.path.exists('data/xena'):
+    os.makedirs('data/xena')
+
+expression = 'https://ndownloader.figshare.com/files/5864859'
+
+if not os.path.exists('data/xena/expression-matrix.tsv.bz2'):
+    request.urlretrieve(expression, os.path.join('data', 'xena',
+                                                 'expression.tsv.bz2'))
+
+mutation = 'https://ndownloader.figshare.com/files/5864862'
+
+if not os.path.exists('data/xena/mutation-matrix.bz2'):
+    request.urlretrieve(mutation, os.path.join('data', 'xena',
+                                               'mutation-matrix.bz2'))
+
+samples = 'https://ndownloader.figshare.com/files/5864865'
+
+if not os.path.exists('data/xena/samples.tsv'):
+    request.urlretrieve(samples, os.path.join('data', 'xena', 'samples.tsv'))
+
+gene_map = 'https://raw.githubusercontent.com/cognoma/cancer-data/master/'\
```

Comment:
A bit dangerous to use versioned data links and then an unversioned URL here. Protip: press `y` on a GitHub page to switch to the versioned URL.

Going to reference https://github.com/cognoma/cancer-data/issues/23 here -- a gene information table that got uploaded to figshare would make your life easier.